GameDataParser/Parsers/MapEntityParser.cs

```diff
@@ -100,386 +67,337 @@ protected override List<MapEntityMetadata> Parse()
-                                if (coordMatch.Success)
-                                {
-                                    metadata.Objects.Add(new MapObject(CoordB.Parse(coordMatch.Value, ", "), int.Parse(weaponId)));
-                                }
-                            }
+                            continue;
                         }
-                        else if (modelData.ContainsKey("mixinMS2Breakable"))
+
+                        try
+                        //TODO: The parser will output errors here, which are non-critical. 
                         {
-                            // "mixinMS2Breakable"  But not "mixinMS2BreakableActor", as in ke_fi_prop_buoy_A01_ or el_move_woodbox_B04_
+                            CoordB coord = CoordB.Parse(coordMatch.Value, ", ");
+                            metadata.Objects.Add(new MapObject(coord, int.Parse(mapProperties.ObjectWeaponItemCode)));
                         }
-                        else if (modelData.ContainsKey("mixinMS2RegionBoxSpawn"))
+                        catch (FormatException)
                         {
-                            // "QR_10000264_" is Quest Reward Chest? This is tied to a MS2TriggerAgent making this object appear.
+                            // ignored
+                            Console.WriteLine($"Format error parsing {mapProperties.ObjectWeaponItemCode} as int");
+                            byte[] bytes = System.Text.Encoding.UTF8.GetBytes(mapProperties.ObjectWeaponItemCode);
+                            Console.WriteLine($"String in bytes: {Convert.ToHexString(bytes)}");
                         }
-                        // Unhandled Items:
-                        // "mixinEventSpawnPointNPC", 
-                        // "mixinMS2Actor" as in "fa_fi_funct_irondoor_A01_"
-                        // MS2RegionSkill as in "SkillObj_co_Crepper_C03_" (Only on 8xxxx and 9xxxxx maps)
-                        // "mixinMS2FunctionCubeKFM" as in "ry_functobj_lamp_B01_", "ke_functobj_bath_B01_"
-                        // "mixinMS2FunctionCubeNIF"
-                        // "MS2MapProperties"->"MS2PhysXProp" that's not a weapon. Standard 
-                    }
-                    /*
-                     * if (Regex.Match(modelName, @"^\d{8}_").Success && Regex.Match(name, @"\d{1,3}").Success)
-                     * {
-                        // Parse non-permanent NPCs. These have no .flat files to supplement them.
-                        string npcListIndex = node.SelectSingleNode("property[@name='NpcList']")?.FirstChild.Attributes["index"].Value ?? "-1";
-                        if (npcListIndex == "-1")
+
+                        catch (OverflowException ex)
                         {
-                            continue;
+                            Console.WriteLine($"Error parsing {mapProperties.ObjectWeaponItemCode} as int: {ex.Message}");
                         }
```

Comment:
we need to print out these?

Reply:
I would leave them in until this error is fixed-
or comment it out